src/api/app/controllers/webui/user_controller.rb

```diff
@@ -42,7 +42,6 @@ def do_login
     end
 
     if user.nil? || (user.state == User::STATES['ichainrequest'] || user.state == User::STATES['unconfirmed'])
```

Comment:
@adrianschroeter btw it looks to me like the state check has no effect. Or am I missing something?